src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -259,13 +262,30 @@ function TTYGViewCtrl($rootScope, $scope, $http, $timeout, $translate, $uibModal
     const loadChats = () => {
         $scope.loadingChats = true;
```

Comment:
I think it would be better if the "loadingChats" and "loadingAgents" flags are not set here. The methods "loadChats" and "loadAgents" return promises, so it would be more appropriate to set the flags where these methods are actually called, if needed. For example, "loadChats" is used in "onExportChat," where showing a loader is unnecessary.